components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -802,11 +802,6 @@ void AtmosphereDriver::run (const int dt) {
   // Make sure the end of the time step is after the current start_time
   EKAT_REQUIRE_MSG (dt>0, "Error! Input time step must be positive.\n");
 
-  // Print current timestamp information
-  if (m_atm_comm.am_i_root()) {
-    std::cout << "Atmosphere step = " << m_current_ts.get_num_steps() << "; model time = " << m_current_ts.get_date_string() << " " << m_current_ts.get_time_string() << std::endl;
-  }
```

Comment:
What is the reason for removing this? I'll be honest, I did think we had something printing out timestep info..

Reply:
Wow, I have no recollection of doing this. Not just in this PR, but ever in my life! I'll revert it.

Reply:
maybe the version of master you built upon was older???  Weird only this was deleted.